README.md

```diff
@@ -194,6 +195,7 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
 * [uxdesign.cc](https://uxdesign.cc/) User Experience, Usability, Product Design
 * [OverAPI.com](http://overapi.com/) Collecting All Cheat Sheets
 * [The list for web designers](https://www.designerslist.info/) A complete toolkit for web designers, wireframes, tutorials, blogs, tools, stock photos, fonts, color schemes, icons...
+* [Dev community](https://dev.to/)
```

Comment:
Please add a small description of this link

Reply:
I have added the description now